platforms/hyperledger-besu/charts/node_tessera/values.yaml

```diff
@@ -140,3 +140,7 @@ labels:
   service: []
   pvc: []
   deployment: []
+
+metrics:
+  enabled:
```

Comment:
Please provide example values for each k:v pair (enabled, port) (check other key:value pairs in this values file)
And I don't think we need the `enabled` field. This could be easily considered by the presence/absence of the metrics port.

Reply:
> The lesser fields we have in our charts (+ placed logically in the values.yaml file, with their example key:value pair), the easier it becomes to use the chart manually (by creating the value files for charts manually) (The easier the charts are to use manually, the easier we can decouple them from ansible in the future)

The idea of having `enabled` field is to simplify the usage for the users, `port` field can be optional (maybe used by advanced users). We can use `helm templates` to have the default value for `port` this way the helm chart can be decoupled from ansible and can be used independently.

Reply:
This sounds cool. Please just provide the example values for each new k:v pair introduced and this is otherwise fine.

Reply:
Sure, I will update the code with examples